sequencers/modeshape-sequencer-ddl/src/test/resources/ddl/create_table_check_constraint.ddl

```diff
@@ -0,0 +1,21 @@
+--  SQL Server / Oracle / MS Access, not named
+--  Note that starting with 5.7 MySQL does not support CHECK constraints 
```

Comment:
I'm not sure this comment is correct. MySQL 5.5-5.7 [all support](http://dev.mysql.com/doc/refman/5.7/en/create-table.html) CHECK constraints, though whether they are _used_ depends on the storage engine.